benches/bank.rs

```diff
@@ -40,7 +40,8 @@ fn bench_process_transaction(bencher: &mut Bencher) {
 
             // Finally, return the transaction to the benchmark.
             tx
-        }).collect();
+        })
+        .collect();
```

Comment:
looks like rust fmt has changed again?

Reply:
oops, rolled back to 1.30.1 and force-pushed, thanks!